graylog2-server/src/test/java/org/graylog2/alerts/AlertConditionTest.java

```diff
@@ -47,6 +47,18 @@
 import static org.mockito.Mockito.when;
 
 public abstract class AlertConditionTest {
+    private abstract class TestAlertConditionFactory implements AlertCondition.Factory {
+        @Override
+        public AlertCondition.Config config() {
+            throw new RuntimeException("This method is not implements.");
```

Comment:
Maybe [`UnsupportedOperationException`](https://docs.oracle.com/javase/8/docs/api/java/lang/UnsupportedOperationException.html) would be more fitting?

Also typo: "implements" &rarr; "implemented"